templates/event.php

```diff
@@ -31,122 +33,163 @@
+								echo '<form class="add-remove-user-as-host" method="post" action="' . esc_url( gp_url( "/events/host/$event_id/$contributor->ID" ) ) . '">';
+								if ( $_attendee->is_host() ) :
+									echo '<input type="submit" class="button is-primary remove-as-host" value="Remove as host"/>';
+								else :
+									echo '<input type="submit" class="button is-secondary convert-to-host" value="Convert to host"/>';
+								endif;
+								echo '</form>';
+							endif;
+							?>
 						</li>
 					<?php endforeach; ?>
 				</ul>
 			</div>
-	<div class="event-details-stats">
-		<h2><?php esc_html_e( 'Stats', 'gp-translation-events' ); ?></h2>
-		<table>
-			<thead>
-			<tr>
-				<th scope="col">Locale</th>
-				<th scope="col">Translations created</th>
-				<th scope="col">Translations reviewed</th>
-				<th scope="col">Contributors</th>
-			</tr>
-			</thead>
-			<tbody>
-			<?php /** @var $row Stats_Row */ ?>
-			<?php foreach ( $event_stats->rows() as $_locale => $row ) : ?>
-			<tr>
-				<td title="<?php echo esc_html( $_locale ); ?> "><a href="<?php echo esc_url( gp_url_join( gp_url( '/languages' ), $row->language->slug ) ); ?>"><?php echo esc_html( $row->language->english_name ); ?></a></td>
-				<td><?php echo esc_html( $row->created ); ?></td>
-				<td><?php echo esc_html( $row->reviewed ); ?></td>
-				<td><?php echo esc_html( $row->users ); ?></td>
-			</tr>
-		<?php endforeach ?>
-			<tr class="event-details-stats-totals">
-				<td>Total</td>
-				<td><?php echo esc_html( $event_stats->totals()->created ); ?></td>
-				<td><?php echo esc_html( $event_stats->totals()->reviewed ); ?></td>
-				<td><?php echo esc_html( $event_stats->totals()->users ); ?></td>
-			</tr>
-			</tbody>
-		</table>
-	</div>
-	<div class="event-projects">
-		<h2><?php esc_html_e( 'Projects', 'gp-translation-events' ); ?></h2>
-		<ul>
-			<?php foreach ( $projects as $project_name => $row ) : ?>
-			<li class="event-project" title="<?php echo esc_html( str_replace( ',', ', ', $row->locales ) ); ?>">
-				<a href="<?php echo esc_url( gp_url_project( $row->project ) ); ?>"><?php echo esc_html( $project_name ); ?></a> <small> to
-				<?php
-				foreach ( explode( ',', $row->locales ) as $_locale ) {
-					$_locale = \GP_Locales::by_slug( $_locale );
-					?>
-					<a href="<?php echo esc_url( gp_url_project_locale( $row->project, $_locale, 'default' ) ); ?>"><?php echo esc_html( $_locale->english_name ); ?></a>
+		<?php endif; ?>
+		<?php if ( ! empty( $attendees ) ) : ?>
+			<div class="event-attendees">
+				<h2><?php esc_html_e( 'Attendees', 'gp-translation-events' ); ?></h2>
+				<small><?php esc_html_e( 'Users without contributions', 'gp-translation-events' ); ?></small>
+				<ul>
+					<?php foreach ( $attendees as $_user ) : ?>
+						<li class="event-attendee">
+							<a href="<?php echo esc_url( get_author_posts_url( $_user->ID ) ); ?>"><?php echo get_avatar( $_user->ID, 48 ); ?></a>
```

Comment:
@trymebytes when this PR is merged, please highlight first-time contributors here. Can we use 🎉 as a avatar overlay (like in the bottom right corner?). Let's also change it in the contributors list so that it's consistent.

Reply:
Sure, I'll come up with a mockup

Reply:
I added it to the contributors' list, because I needed to fix a conflict merge from trunk. After this, I added it to the attendees' list [here](https://github.com/WordPress/wporg-gp-translation-events/pull/200/commits/eb99f44120e3cfb6e4ac9266e99c9eaa508c90fe).